web/themes/ventuno/src/scss/components/_homepage.scss

```diff
@@ -0,0 +1,77 @@
+// Temporarily hide homepage title
+#block-ventuno-pagetitle {
+  display: none;
+}
+
```

Comment:
Is it OK for you if I hide the pagetitle block on the homepage? Then we don't need this workaround.

Reply:
Yes perfect!